client/galaxy/scripts/layout/page.js

```diff
@@ -30,6 +30,7 @@ var View = Backbone.View.extend({
                 Utils.setWindowTitle();
                 view.allow_title_display = true;
             }
+            view.active_tab && Utils.setActiveTab(view.active_tab);
```

Comment:
Looks like a good start!  Instead of this shortcut, do you mind using the explicit conditional form?  We're trying to remove these as we overhaul the client code.

```
if (view.active_tab) {
    Utils.setActiveTab(view.active_tab);
}
```